test-sbt/shared/src/main/scala/zio/test/sbt/ZTestEvent.scala

```diff
@@ -19,15 +21,15 @@ object ZTestEvent {
     executedSpec: ExecutedSpec[L, E, S],
     fullyQualifiedName: String,
     fingerprint: Fingerprint
-  ): Seq[ZTestEvent] = {
-    def loop(executedSpec: ExecutedSpec[String, E, S]): Seq[ZTestEvent] =
-      executedSpec.caseValue match {
-        case Spec.SuiteCase(_, executedSpecs, _) => executedSpecs.flatMap(loop)
-        case Spec.TestCase(label, result) =>
+  ): UIO[Seq[ZTestEvent]] =
+    executedSpec.mapLabel(_.toString).fold[UIO[Seq[ZTestEvent]]] {
+      case Spec.SuiteCase(_, results, _) =>
+        results.flatMap(UIO.collectAll(_).map(_.flatten))
```

Comment:
collectAll makes a choice of sequentiality—intentional?

Reply:
My thought was that since we are already dealing with an `executedSpec` here the potentially compute intensive work of running the test has already been done so we don't need parallelism here.